pkg/controller/installation/validation.go

```diff
@@ -331,6 +331,30 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 				return fmt.Errorf("spec.calicoNetwork.containerIPForwarding is supported only for Calico CNI")
 			}
 		}
+
+		type TuningSpec struct {
+			Sysctl []operatorv1.Sysctl `json:"sysctl,omitempty"`
+			Type   string              `json:"type"`
+		}
```

Comment:
Sorry I hadn't mentioned this before because I had suggested all the changes in this file be dropped but now that we're keeping the validation, this code is unused here so should be removed. I think it probably got copied from somewhere else and missed that it was not used.

Reply:
you're right, i missed it. resolved